patchwork-user-profiles.md

```diff
@@ -1,299 +0,0 @@
-# Patchwork: User profiles
```

Comment:
i'm intending to illustrate having a clear set of well-known objectives / principles that steer the direction of the project and are used as a measure of success. [another example of this from Enspiral](http://joshuavial.com/mastermind-setting-strategy-together/), feel free to ignore.